app/assets/stylesheets/koi/components/_index-table.scss

```diff
@@ -62,15 +62,30 @@ $row-height: 48px !default;
       max-height: 3rem;
       padding: 0;
     }
+  }
 
-    &.number {
+  td {
+    &.katalyst--tables-column-currency,
+    &.katalyst--tables-column-number {
```

Comment:
I'm not a fan of these super-long names. What's the thinking here?

I think this hurts reusability. It's also confusing because these are koi tables not katalyst tables.